index.html

```diff
@@ -21,7 +21,6 @@
 <head>
     <meta charset="UTF-8">
     <title>macOSNotifJS</title>
-    <link href="plugin/res/macOSNotif.fonts.min.css" rel="stylesheet" type="text/css"/>
```

Comment:
This doesn't appear to have been replaced?

Reply:
Just noticed your comment about this, is there a way to get this working or not? On the package site it serves this page, so having it as a working demo would be nice.